app/utils.ts

```diff
@@ -14,7 +14,7 @@ export function safeRedirect(
   to: FormDataEntryValue | string | null | undefined,
   defaultRedirect: string = DEFAULT_REDIRECT
 ) {
-  if (!to || typeof to !== "string") {
+  if (!to || typeof to !== "string" || to === "") {
```

Comment:
Weird, I would have thought the first conditional in this if statement would suffice for empty strings.

Reply:
Not weird at all, this is what you get from a manager who doesn't code everyday 😉  I'll remove that empty string conditional.